.github/actions/deploy/action.yaml

```diff
@@ -25,7 +26,7 @@ runs:
         repository: Cardinal-Cryptography/aleph-apps
         token: ${{ inputs.github_token }}
         path: "aleph-apps"
-        ref: ${{ inputs.environment }}
+        ref: ${{ inputs.environment == 'devnet' && 'main' || inputs.environment }}
```

Comment:
`main` or `master`?

Reply:
We definitely use `master` here @Marcin-Radecki

Reply:
But this is checkout `ref` for our apps repo that has `main` default branch. So it seems correct?

![image](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/assets/3909333/968e09aa-31b9-4754-85f7-9fc0c5dd6b2f)

Reply:
sorry, I misled us, my bad :see_no_evil: